### ISSUE TITLE
Peer Review - Nehal Rawat (nr338)

### ISSUE THREAD

**Issue by nehalrawat** [laurensedita/4741project#3]
This project aims to analyze and understand US unemployment trends over different demographics, and predict which factors such as age, gender, race, education, job training, occupation type, geographical area, health, drug use, and criminal record lead to long term unemployment. 

Three things I like about the proposal:

1) I think the topic of analyzing unemployment trends, specifically using youth data is highly relevant. A greater understanding of this can help boost employment rates and help individuals develop while they are still young, thereby mitigating the problem from its root.

2) The dataset is comprehensive and covers many possible factors. In addition to the main factors listed above, it also covers military involvement. Therefore, the project has good scope and flexibility in choosing which factors to focus on.

3) The proposal very clearly indicates how the data is segregated, such as how it is divided into subsets, what time period the data is derived from, and how many entries are available for each subgroup. 

Three areas for improvement:

1) It would be helpful to outline a specific question you would like to answer. While the proposal introduces the scope of the project well, it does not clearly indicate what the desired outcome is or what specific predictions will be made to understand long term unemployment in the US.

2) The current dataset has many features, which might lead to overfitting and less accurate test set outcomes. You could use dimensionality reduction to look at a smaller set of features. 

3) The proposal mentions that the sample members were surveyed once every year from 1979 to 1994, and then biennially from 1994 to 2014. How will you deal with this inconsistency? An important aspect to analyze is how unemployment trends change over time, and standardizing this would lead to a more statistically sound model. 

This project seems really interesting, and is very relevant!
